.github/workflows/ci.yml

```diff
@@ -28,9 +28,11 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        scala: [2.13.10, 2.12.17]
+        scala: [3.2.1, 2.13.10, 2.12.17]
         java: [corretto@11, corretto@8]
         exclude:
+          - scala: 3.2.1
```

Comment:
looks like this is no longer needed, after Java 8 deprecation PR is merged

Reply:
This file is now generated from `build.sbt` and CI fails if it detects it is not in sync.
When we'll update the branch with changes in main, we'll just need to run `sbt githubWorkflowGenerate` to have this updated.